### ISSUE TITLE
callProcess: rm "/tmpcqzdatzfva.mp4" (exit 1): failed

### ISSUE THREAD

**Issue by muralikodali** [rberenguel/glancer#12]
glancer fails with following error without creating html file :  
Looking for video in https://youtu.be/c8BaLpd69QI (this may take a while)
The video is titled 'Old PC Upgrade #2: SSD Boot Drive'
Seems like the video is in https://www.youtube.com/watch?v=c8BaLpd69QI
Downloaded video to /tmpcqzdatzfva(.mp4|en.vtt)
Generating still images from video (this may take a while)
Generated images
rm: cannot remove '/tmpcqzdatzfva.mp4': No such file or directory
glancer: callProcess: rm "/tmpcqzdatzfva.mp4" (exit 1): failed

i think there should be slash after /tmp i.e: /tmp/  to remove file

**Comment by rberenguel** [comment]
What operating system are you using @muralikodali ? This is probably a duplicate of #9 so I'm aware, will fix as soon as I have some time available. In the link where it was reported, the fix was added so feel free to PR it if you get around making the changes. Thanks for reporting and using it!

**Comment by muralikodali** [comment]
My OS is Arch Linux

**Comment by rberenguel** [comment]
@muralikodali can you try now? I have stopped constructing paths manually and instead I'm relying on the filepath library (I wonder why I didn't from the beginning). Thanks!

**Comment by muralikodali** [comment]
failing with error as shown below :  

![image](https://user-images.githubusercontent.com/11482766/108025381-63da2c00-704c-11eb-903e-2a1aefc6bc08.png)

**Comment by ber2** [comment]
It seems that the recent changes do write files to `/tmp/blah.en.vtt` (the log message misses a `/` however).

The problem is when, further along, an attempt is made at reading the path; the `/` is missing and it looks for `/tmpblah.en.vtt`.

Should be fixable by adding an `</>` somewhere, perhaps in the `start` function in `Main.hs`. 😉

**Comment by rberenguel** [comment]
![IMG_1251](https://user-images.githubusercontent.com/2410938/108327807-1f759a00-71cc-11eb-87ef-7b109a183c8b.GIF)

**Comment by muralikodali** [comment]
it is now successfully building html page.
html page layout is good. but the text part is hard to parse **with align:start position:0%** seen on each sentence.

![image](https://user-images.githubusercontent.com/11482766/108663467-a55c5280-74f6-11eb-9b43-788b35bfd27e.png)

**Comment by rberenguel** [comment]
Thanks @muralikodali I'll try to fix this today. Captions contain these markup commands, they are not added by glancer.

**Comment by rberenguel** [comment]
@muralikodali fixed, it was easier than expected (although I hope Youtube does not start adding extra markup to captions!)
